### PR TITLE
feat(client): add new coder survey link to learn

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

If the space looks to large, it looks less so when there is a _Go to current challenge_ button.
image

### CODE REVIEW

File: client/i18n/locales/chinese-traditional/translations.json [hunk 1]
```diff
@@ -286,7 +286,8 @@
       "preview": "預覽"
     },
     "help-translate": "我們仍然在翻譯以下證書。",
-    "help-translate-link": "幫助我們翻譯。"
+    "help-translate-link": "幫助我們翻譯。",
+    "coder-survey": "<0>Take the 2021 New Coder Survey<0> (it takes about 10 minutes – do it for science)"
```
Comment (nhcarrigan) [inline client/i18n/locales/chinese-traditional/translations.json:290]:
```suggestion
    "coder-survey": "<0>Take the 2021 New Coder Survey</0> (it takes about 10 minutes – do it for science)"
```

File: client/i18n/locales/chinese/translations.json [hunk 2]
```diff
@@ -286,7 +286,8 @@
       "preview": "预览"
     },
     "help-translate": "我们仍然在翻译以下证书。",
-    "help-translate-link": "帮助我们翻译。"
+    "help-translate-link": "帮助我们翻译。",
+    "coder-survey": "<0>Take the 2021 New Coder Survey<0> (it takes about 10 minutes – do it for science)"
```
Comment (nhcarrigan) [inline client/i18n/locales/chinese/translations.json:290]:
```suggestion
    "coder-survey": "<0>Take the 2021 New Coder Survey</0> (it takes about 10 minutes – do it for science)"
```

File: client/i18n/locales/english/translations.json [hunk 3]
```diff
@@ -286,7 +286,8 @@
       "preview": "Preview"
     },
     "help-translate": "We are still translating the following certifications.",
-    "help-translate-link": "Help us translate."
+    "help-translate-link": "Help us translate.",
+    "coder-survey": "<0>Take the 2021 New Coder Survey<0> (it takes about 10 minutes – do it for science)"
```
Comment (nhcarrigan) [inline client/i18n/locales/english/translations.json:290]:
```suggestion
    "coder-survey": "<0>Take the 2021 New Coder Survey</0> (it takes about 10 minutes – do it for science)"
```

File: client/i18n/locales/espanol/translations.json [hunk 4]
```diff
@@ -286,7 +286,8 @@
       "preview": "Vista"
     },
     "help-translate": "Todavía estamos traduciendo las siguientes certificaciones.",
-    "help-translate-link": "Ayúdanos a traducir."
+    "help-translate-link": "Ayúdanos a traducir.",
+    "coder-survey": "<0>Take the 2021 New Coder Survey<0> (it takes about 10 minutes – do it for science)"
```
Comment (nhcarrigan) [inline client/i18n/locales/espanol/translations.json:290]:
```suggestion
    "coder-survey": "<0>Take the 2021 New Coder Survey</0> (it takes about 10 minutes – do it for science)"
```

File: client/i18n/locales/italian/translations.json [hunk 5]
```diff
@@ -286,7 +286,8 @@
       "preview": "Anteprima"
     },
     "help-translate": "Stiamo ancora traducendo le seguenti certificazioni.",
-    "help-translate-link": "Aiutaci con le traduzioni..."
+    "help-translate-link": "Aiutaci con le traduzioni...",
+    "coder-survey": "<0>Take the 2021 New Coder Survey<0> (it takes about 10 minutes – do it for science)"
```
Comment (nhcarrigan) [inline client/i18n/locales/italian/translations.json:290]:
```suggestion
    "coder-survey": "<0>Take the 2021 New Coder Survey</0> (it takes about 10 minutes – do it for science)"
```

File: client/i18n/locales/portuguese/translations.json [hunk 6]
```diff
@@ -286,7 +286,8 @@
       "preview": "Pré-visualizar"
     },
     "help-translate": "Ainda estamos traduzindo as certificações a seguir.",
-    "help-translate-link": "Ajude-nos a traduzir."
+    "help-translate-link": "Ajude-nos a traduzir.",
+    "coder-survey": "<0>Take the 2021 New Coder Survey<0> (it takes about 10 minutes – do it for science)"
```
Comment (nhcarrigan) [inline client/i18n/locales/portuguese/translations.json:290]:
```suggestion
    "coder-survey": "<0>Take the 2021 New Coder Survey</0> (it takes about 10 minutes – do it for science)"
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Comment by ShaunSHamilton** [comment]
@QuincyLarson Should the link open in a new tab?

**Review by naomi-lgbt — APPROVED** [review]

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Review by QuincyLarson — APPROVED** [review]

**Pull request merged**